Replicator/tests/data/cert/cert.pem

```diff
@@ -0,0 +1,17 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Where is this cert used?

Reply:
I believe it's the same as the leaf certificate (the first cert in sg_cert.pem). It was generated by the scripts/gen_cert.sh script